charts/open-zaak/values.yaml

```diff
@@ -7,29 +7,32 @@ replicaCount: 1
 image:
   repository: openzaak/open-zaak
   pullPolicy: IfNotPresent
-  tag: 1.4.0
+  tag: 1.5.0
 
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+initContainers:
+  volumePerms: true
+
 serviceAccount:
   create: true
   annotations: {}
   name: ""
 
 podAnnotations: {}
 
-podSecurityContext: {}
-  # fsGroup: 2000
+podSecurityContext:
+  fsGroup: 1000
 
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+securityContext:
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: false
+  runAsNonRoot: true
+  runAsUser: 1000
```

Comment:
I think this is not "required". When USER is specified in the Dockerfile, you can leave this empty.